dex-it/src/test/scala/com/wavesplatform/it/sync/api/ws/WsOrderBookStreamTestSuite.scala

```diff
@@ -416,6 +416,33 @@ class WsOrderBookStreamTestSuite extends WsSuiteBase {
       }
     }
 
+    "get an error when subscribing to deleted orderbook" in {
+      val seller = mkAccountWithBalance(100.waves -> Waves)
+      val IssueResults(issueTx, _, asset) = mkIssueExtended(seller, "new asset", 1000.asset8)
+      val assetPair = AssetPair(asset, Waves)
+
+      broadcastAndAwait(issueTx)
+      dex1.api.place(mkOrderDP(seller, assetPair, SELL, 100.asset8, 5.0))
```

Comment:
can this test be flacky without awaiting the order at dex?

Reply:
We found out that if we place an order and wait for status "Accepted", then we don't need to await it - `dex1.api.place` behaves the same as `dex1.api.place` and `waitForOrderStatus`